financial-templates-lib/logger/PagerDutyTransport.js

```diff
@@ -17,6 +17,9 @@ module.exports = class PagerDutyTransport extends Transport {
 
   async log(info, callback) {
     try {
+      // If the message has markdown then add it and the bot-identifer field. Else put the whole info object as a string
+      const logMessage = info.mrkdwn ? info.mrkdwn + info["bot-identifier"] : JSON.stringify(info);
```

Comment:
unrelated to the bot fix but I realized that if the message has markdown then it does not identify itself in pagerduty. This additional string will ensure that you can see the bot that created the log message.